### DISCUSSION TITLE
Lab Assignment #2

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#12] [Assignments]
## Due 4/13 before class 

Please use this thread to post your [lab assignment's #2](https://albertkun.github.io/22S-ASIAAM-191A/assignments/week2/lab_assignment/) link, which is the GitHub pages URL. [Click here for a reminder on how to get it](https://albertkun.github.io/22S-ASIAAM-191A/labs/week1/3/#using-github-pages).

### Note 
If your map is not on the main `index.html` page, be sure to submit that sub-page's link instead:

#### For example:
https://albertkun.github.io/22s-asiaam-191a-sample/week2/map.html

If you only have one page, then you can just submit the `index.html` link as per usual.

**Comment by emmashanson** [comment]
https://emmashanson.github.io/22s-asianam-191a-labs/week%202/index.html

**Comment by shubhgupta2018** [comment]
https://shubhgupta2018.github.io/22s-191a-assignments/week-2/index.html

**Comment by antonyblin** [comment]
https://antonyblin.github.io/aas191a-assignments/week2/food.html

**Comment by NishuKrish** [comment]
https://nishukrish.github.io/AsianAm191A-Nish/week2/index.html

**Comment by daikis12** [comment]
https://daikis12.github.io/22s-191a-assignments/week2/index.html

**Comment by ntran27** [comment]
https://ntran27.github.io/22s-asiaam-191a-labs/week2/index.html

**Comment by angelineeow** [comment]
https://angelineeow.github.io/22s-asiaam-191a-labs/week2/more_about_me.html

**Comment by ananyakodali17** [comment]
https://ananyakodali17.github.io/asian-am-191a-labs/week2/

**Comment by hajovel** [comment]
https://hajovel.github.io/211S-191A-Assignments/week2/map

**Comment by stefuntran** [comment]
https://stefuntran.github.io/22s-asiaam-191a-labs/week%202/index.html

**Comment by bittertang** [comment]
https://bittertang.github.io/22s-asiaam-191a-labs/week%202/

**Comment by willa-21** [comment]
https://willakurland.github.io/aa191-labs/Week-2/index.html

**Comment by XiongHer** [comment]
https://xiongher.github.io/22s-asiaam-191a-labs/week2/index.html

**Comment by kris10fu** [comment]
https://kris10fu.github.io/22s-asiaam-191a-labs/week2/map.html

**Comment by penelopelamb** [comment]
https://penelopelamb.github.io/22s-aas-191a-labs/week2/index.html

**Comment by kericsp** [comment]
https://kericsp.github.io/aa191-labs/week2/index.html (if anyone can help me get my images to show up in the markers I would appreciate you so much! 🥺)

**Reply by albertkun** [reply]
Hello! In your code where you create your marker function you need to specify the `<img src={$image path}></img> `

Right now you have it so it just adds the image text into the `<img>` element.

**Reply by kericsp** [reply]
Hmm, it's still not working, and if i take out the <img> tag then my captions don't show up either 😭

**Reply by kericsp** [reply]
Does it count for extra credit if the images only show up correctly in firefox and not chrome :')

**Reply by albertkun** [reply]
@kericsp hahah, sure! its the attempt that matters :)

**Reply by albertkun** [reply]
remember to save the images locally though!!! do not hot link!!

**Comment by joonyloony** [comment]
https://joonyloony.github.io/22s-191a-assignments/week2/index.html

**Comment by codingdianat** [comment]
https://codingdianat.github.io/aa191-assignments/week1/

**Comment by spolcharoen** [comment]
https://spolcharoen.github.io/22s-asiaam-191a-labs/week2/index.html

**Comment by ashley-lu** [comment]
https://ashley-lu.github.io/s22-asiaam-191a-assignments/week2lab/index.html

**Comment by kaittli** [comment]
https://kaittli.github.io/22s-asiaam-191a-labs/week2/index.html

:((

**Comment by lindseymardona** [comment]
https://lindseymardona.github.io/22s-asiaam-191a-labs/week2/index.html
